extra/1-radio-stations.js

```diff
@@ -25,11 +42,17 @@
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
+function isRadioStation(num) {
```

Comment:
This function declaration causes an exception which breaks the otherwise good code. You can't declare two functions with the same name in the same scope, and in this case, an isRadioStation function already exists, so it'll throw a SyntaxError